LICENSE

```diff
@@ -1,21 +1,28 @@
-MIT License
 
-Copyright (c) 2020 Adam Tyson
+Copyright (c) 2023, Niko Sirmpilatze
```

Comment:
Don't worry about changing this, but we should come up with a standard thing to put here, probably UCL.